bitrise.yml

```diff
@@ -1132,7 +1132,7 @@ workflows:
         is_always_run: true
     - xcode-test@2:
         inputs:
-        - scheme: Fennec_Enterprise_SmokeXCUITests
```

Comment:
I think we still need to have a scheme here, now: Fennec_Enterprise_XCUITest.

Reply:
Ah they still need a scheme that's right. Thanks.